crates/bevy_app/src/app_builder.rs

```diff
@@ -273,13 +427,44 @@ impl AppBuilder {
         self
     }
 
+    /// Adds a group of plugins
+    ///
+    /// Bevy plugins can be grouped into a set of plugins. By default
+    /// bevy provides a few lists of plugins that can be used to kickstart
+    /// the development.
+    ///
+    /// Current plugins offered are [`DefaultPlugins`] and [`MinimalPlugins`].
+    ///
+    /// ## Example
+    /// ```
+    /// # use bevy_app::prelude::*;
+    /// #
+    /// App::build();
+    ///     //.add_plugins(MinimalPlugins)
```

Comment:
Why is this commented out?

Reply:
Not OP, but their introductory comment mentioned that line pulling in too many dependencies.

Reply:
Thanks for pointing me to that.